README.md

```diff
@@ -38,12 +38,52 @@
 
 
 
-# 📖 [Portofolio form validation] <a name="about-project"></a>
+ preserve-data-in-the-browser
+# 📖 [ preserve data in the browser] <a name="about-project"></a>
+
+
 
 > **This is the final modification of the portfolio website project**
+ preserve-data-in-the-browser
+**This is not a  full responsive  website. It's just a  web look alike based on the figma design we were provided.**
+**The form section is constructed so that they will be no submission if the email and full name inputs are not available.**
+**The data insterted in the form are stored in the browser locally**
+
+**[ preserve_data_in_the_browser]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to remove all the comments and guides in your readme file to make it more professional :joy: and please try to remove unused sections